dev/graphdb_setup.sh

```diff
@@ -128,12 +128,12 @@ echo "Environment variables have been set from ${ENV_FILE_PATH}."
 
 # Extract just the database name
 DB_NAME="${NB_GRAPH_DB#repositories/}"
-NB_GRAPH_PORT_HOST=${NB_GRAPH_PORT_HOST:-7200}
+NB_GRAPH_PORT=${NB_GRAPH_PORT:-7200}
 
 # Get the directory of this script to be able to find the data-config_template.ttl file
 SCRIPT_DIR=$(dirname "$0")
 
-echo "The GraphDB server is being accessed at http://localhost:${NB_GRAPH_PORT_HOST}."
+echo "The GraphDB server is being accessed at http://localhost:${NB_GRAPH_PORT}."
```

Comment:
```suggestion
echo "The GraphDB server is being accessed at inside the graphDB container http://localhost:${NB_GRAPH_PORT}."
```

My head is spinning a bit about what `localhost` can mean in all of these scenarios. Idea: since we really only expect this script to be run as part of the single `docker-compose.yml` recipe in the future - should we update these echos accordingly?

Reply:
e.g. to understand that `localhost` is correct here you need to remember that
- this is running in a container, not host
- that it is running in the exact container that is running graphDB (otherwise you'd have to use `graph`, not `localhost`)
- and that by `localhost`, we're referring to the container itself, not the host machine

quite a bit of things to remember